website/docs/docs/dbt-cloud/access-control/self-service-permissions.md

```diff
@@ -18,3 +16,21 @@ dbt Cloud supports two different permission sets to manage permissions for self-
 | Manage billing | ❌ | ✅ |
 | Manage team permissions | ❌ | ✅ |
 | Invite Owners to the account | ❌ | ✅ |
+
+## Read Only vs. Developer License Types
+
+Users configured with Read Only license types will experience a restricted set of permissions in dbt Cloud. If a user is associated with an member permission set and a Read Only seat license, then they will only have access to what a Read-Only seat allows. See [Seats and Users](cloud-seats-and-users) for more information on the impact of licenses on these permissions.
+
+## Owner and Member Groups in dbt Cloud Enterprise
+
+By default, new users are added to the Member and Owner groups when they onboard to dbt Cloud. This is necessary for account onboarding but can create confusion when initially setting up SSO and RBAC for dbt Cloud Enterprise accounts as described in the [Enterprise Permissions](enterprise-permissions) guide. Owner and Member groups are **account level** groups, so their permissions override any project-level permissions you wish to apply.
+
+We recommend the following steps be taken after onboarding Administrative users and before adding less priveleged users to a dbt Cloud Enterprise account.
```

Comment:
Sorry to be nitpicky here but can we say:

> After onboarding Adimstrative users, complete the following steps after SSO and RBAC has been successfully created


We really don't want them to delete the groups before RBAC has been setup or else...they will have no permissions at all to set up RBAC. 😓

Reply:
I should also note that we do mention deleting the Owner and Member group in the RBAC loom video. Perhaps we should also link to this section on the Enterprise permissions page.

Reply:
@amychen1776 Good thinking on spelling out that part of the process. Thoughts on adding another step to "add an Admins group for your privileged users" to make sure those folks can do the other group management things described below?

Reply:
I like the update; do you think it would be better if we just spell out that they need to create an administrator group with the Account Admin permission set? I think there should be no account with no account admins on it so it makes to just say, this is what you need to do .

Reply:
did this, good call!